lib/client/api.go

```diff
@@ -326,7 +326,9 @@ func (tc *TeleportClient) SSH(command []string, runLocally bool) error {
 }
 
 // Join connects to the existing/active SSH session
-func (tc *TeleportClient) Join(sessionID session.ID) (err error) {
+func (tc *TeleportClient) Join(sid string) (err error) {
```

Comment:
you are not supposed to pass any random strings there any ways, you'd need to use session.NewID()